### ISSUE TITLE
Could you share your parameters?

### ISSUE THREAD

**Issue by vincent-roye** [Rajat-dhyani/Stock-Price-Predictor#3]
Could you please give the parameters that you used for the improved LSTM model?

- unroll_length

- model.compile(optimizer='?',loss='?')

All the best and thanks again for your work.
